media-video/ffmpeg/ffmpeg-9999.ebuild

```diff
@@ -430,6 +445,11 @@ multilib_src_compile() {
 				emake V=1 tools/${i}
 			fi
 		done
+
+		if use chromium; then
+			einfo "Compiling for Chromium"
+			emake -C ../chromium V=1 libffmpeg
```

Comment:
wont that overwrite $BUILD_DIR libraries and objects with the extra chromium flags and thus break ABI ?

Reply:
No? We configured in a separate `chromium` directory alongside the other arch-based ABIs and we're changing to that directory here with `-C`.

Reply:
you're right; I would still feel better if you'd pushd/popd there instead :)